packages/vite-plugin-svelte/src/utils/constants.js

```diff
@@ -6,14 +8,16 @@ export const SVELTE_IMPORTS = [
 	'svelte/animate',
 	'svelte/easing',
 	'svelte/internal',
-	'svelte/internal/disclose-version',
-	'svelte/motion',
 	'svelte/ssr',
 	'svelte/store',
 	'svelte/transition',
 	'svelte'
 ];
 
+if (!isSvelte5) {
+	// TODO add back to list above once it's part of svelte5
+	SVELTE_IMPORTS.push('svelte/internal/disclose-version', 'svelte/motion');
+}
```

Comment:
It seems like the v2 branch does this instead:

https://github.com/sveltejs/vite-plugin-svelte/blob/ec4c530125b446e1d8b5f5f84e62aafee6275320/packages/vite-plugin-svelte/src/utils/constants.js#L17-L20

Which I think means we can revert this part and put it in `SVELTE_IMPORTS` always.

Reply:
yes, disclose version has been added to 5 after this